stumpy/stump.py

```diff
@@ -154,11 +154,15 @@ def _calculate_squared_distance_profile(m, QT, μ_Q, σ_Q, M_T, Σ_T):
     denom = m * σ_Q * Σ_T
     denom[denom == 0] = 1e-10  # Avoid divide by zero
     D_squared = np.abs(2 * m * (1.0 - (QT - m * μ_Q * M_T) / denom))
+    # In fastmath mode checking for np.nan is not possible, so we use infs instead
```

Comment:
Can you remind me where else we are checking for `np.nan`? Should this comment be here?

Reply:
In my first implementation I was working with NaN instead of inf until I realized that we run into issues in fastmath mode. I removed the comment